astropy/constants/constant.py

```diff
@@ -75,10 +75,33 @@ def wrapper(self, *args, **kwargs):
 
 
 class Constant(Quantity, metaclass=ConstantMeta):
-    """A physical or astronomical constant.
+    """
+    A physical or astronomical constant.
 
     These objects are quantities that are meant to represent physical
     constants.
+
+    Parameters
+    ----------
+    abbrev : str
+        A typical ASCII text abbreviation of the constant, generally
+        the same as the Python variable used for this constant.
+    name : str
+        Full constant name.
+    value : numbers.Real
+        Constant value. Note that this should be a bare number, not a
+        quantity.
+    unit : str
+        String representation of the constant units.
+    uncertainty : numbers.Real
```

Comment:
```suggestion
    uncertainty : `numbers.Real`
```

Reply:
I don't think back-ticks are necessary here - numpydoc may take care of it. Probably good to check how it rendered.

Reply:
The back-ticks are not needed, see https://astropy--13282.org.readthedocs.build/en/13282/api/astropy.constants.Constant.html#astropy.constants.Constant

Reply:
Oooo, that is magical. @Cadair would be happy. Thanks for the clarification!